.github/workflows/build.yml

```diff
@@ -19,30 +18,22 @@ on:
       - master
       - feature/*
     tags:
-      - '*'
+      - "*"
   workflow_dispatch:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, windows-2022, macos-12]
-    name: 'Build'
+        os: [ubuntu-20.04, windows-2022, macos-13]
```

Comment:
I tried moving us to macos-14 but ran into OOM when publishing with ReadyToRun enabled. The [arm64 MacOS runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) only have 7GB of RAM as opposed to the other platforms having 14GB or more.